### PR TITLE
Add docsearch configs for Diem developer sites

### PR DESCRIPTION
**Pull request by JoelMarcey**
# Pull request motivation(s)

Add configs for the new Diem developer sites

### What is the current behaviour?

No search results

### What is the expected behaviour?

Search results on the sites

### CODE REVIEW

File: configs/diem_developer_website.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "text": "[class^='docItemContainer_'] p, [class^='docItemContainer_'] li"
+  },
+  "selectors_exclude": [
+    ".hash-link"
+  ],
+  "conversation_id": [
+    "1176378229"
+  ],
+  "nb_hits": 989
+}
```
Comment (JoelMarcey) [inline configs/diem_developer_website.json:32]: I use the same values here as another config - https://github.com/algolia/docsearch-configs/blob/master/configs/libra-website.json

I hope that is ok?

File: configs/diem_improvement_proposals.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "text": "article p, article li"
+  },
+  "selectors_exclude": [
+    ".hash-link"
+  ],
+  "conversation_id": [
+    "1129563849"
+  ],
+  "nb_hits": 187
+}
```
Comment (JoelMarcey) [inline configs/diem_improvement_proposals.json:28]: I use the same values here as another config - https://github.com/algolia/docsearch-configs/blob/master/configs/libra-lips.json

I hope that is ok?

File: configs/diem_developer_website.json (hunk 1 above)
Comment (shortcuts) [reply configs/diem_developer_website.json]: All good, I updated it to make sure it match your website structure.

File: configs/diem_improvement_proposals.json (hunk 2 above)
Comment (shortcuts) [reply configs/diem_improvement_proposals.json]: All good, I updated it to make sure it match your website structure.

**Comment by shortcuts** [comment]
Hi @JoelMarcey,

Thanks for the PR.

I've updated the `lvl0` selector and will send you the credentials once deployed!

Feel free to @ me if there's any issue related to it

**Comment by JoelMarcey** [comment]
@shortcuts Thank you so much for correcting and merging this. I really appreciate it.

Do you have an idea of when we should start seeing results on the site -- for example results from the search box on https://developers.diem.com ?

**Comment by shortcuts** [comment]
@JoelMarcey Crawl task has been started and should already been effective or will be in the next 24 hours max!

I'm sending your credentials (apiKey/indexName) to your mail address so make sure to also update it on your side and also solve your ticket regarding Libra account access

**Pull request merged**